Gemfile

```diff
@@ -1,23 +1,18 @@
 source 'https://rubygems.org'
 
-gem 'addressable', '2.3.5'      # pin to avoid RDF bug
-gem 'dor-services', '~> 4.8'
+# General ruby dependencies
+gem 'pry', '~> 0.10.0'           # for bin/console
+gem 'slop', '~> 3.5.0'           # for bin/run_robot
+gem 'rake', '~> 10.3'
+
+# DLSS gems
+gem 'dor-services', '~> 5.3'
 gem 'lyber-core', '~> 3.2', '>=3.2.4'
 gem 'robot-controller', '~> 1.0' # requires Resque
-gem 'pry', '~> 0.10.0'          # for bin/console
-gem 'slop', '~> 3.5.0'          # for bin/run_robot
-gem 'rake', '~> 10.3.2'
 
-group :development do
-  if File.exist?(mygems = File.join(ENV['HOME'], '.gemfile'))
-    instance_eval(File.read(mygems))
-  end
+group :development, :test do
   gem 'rspec'
-  gem 'awesome_print'
-  gem 'debugger', :platform => :ruby_19
   gem 'yard'
-  gem 'capistrano', '~> 3.2.1'
-  gem 'capistrano-bundler', '~> 1.1'
-  gem 'lyberteam-capistrano-devel', '~> 3.0'
+  gem 'dlss-capistrano'
   gem 'holepicker', '~> 0.3', '>= 0.3.3'
 end
```

Comment:
I might agree, but not really the point here.  `test` didn't even exist previously.  I do agree re: `deployment`, in particular since `lyberteam-capistrano-devel` doesn't exist anymore, seems like limiting the scope as tightly as possible for those would be an improvement.  Not the focus of this bump.